params/protocol_params.go

```diff
@@ -159,7 +159,6 @@ const (
 	AdditionalGasForNonGoldCurrencies uint64 = 3*ExpectedGasForCreditToTransactions + ExpectedGasForDebitFromTransactions + ExpectedGasToReadErc20Balance
```

Comment:
Are these parameters used anymore?
`AdditionalGasForNonGoldCurrencies`, `ExpectedGasForDebitFromTransactions`, `ExpectedGasForCreditToTransactions`, `ExpectedGasToReadErc20Balance`?

Reply:
They are only used as defaults before the contract is deployed.

Reply:
They are only used as defaults before the contract is deployed.

Reply:
In that case, would you mind deleting the above variables and adding this:

```
	// Default intrinsic gas cost of transactions paying for gas in alternative currencies.
        IntrinsicGasForAlternativeGasCurrency uint64 = 150000
```

And moving the `MaxGas*` variables down with the other `MaxGas*` variables?